### PR TITLE
Changed asterisk on coupon creation form page.

### PR DESCRIPTION
**Pull request by vkaracic**
https://openedx.atlassian.net/browse/SOL-1835
Partially addressed in https://github.com/edx/ecommerce/pull/836 but I missed a couple of fields. This PR contains the last changes for the workflow. Screenshots below.

screenshot from 2016-07-04 14-00-53
screenshot from 2016-07-04 14-01-32

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @vkaracic! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fecommerce&number=840

**Comment by vkaracic** [comment]
@srpearce @roderickmorales please review this.

**Comment by roderickmorales** [comment]
👍

**Comment by srpearce** [comment]
👍  for this. How do I request a change to the error message? I would recommend deleting "Error!" (which sounds a bit abrupt) and just leaving it as "Please complete all required fields."

**Comment by vkaracic** [comment]
That part is used by both the coupon form and course form, and I'm not sure if we're allowed to change it on the course side as well.

**Comment by vkaracic** [comment]
Disregard that. Here's the latest:
screenshot from 2016-07-07 08-08-46

File: ecommerce/static/js/views/coupon_form_view.js [hunk 1]
```diff
@@ -527,15 +529,39 @@ define([
...
                 e.stopPropagation();
                 window.open(e.currentTarget.href);
             },
 
+
+            /**
+             * Override default renderAlert to display a custom message.
+             */
+            renderAlert: function (level, message) {
+                if (!this.model.isValid(true)) {
```
Comment (marjev) [inline ecommerce/static/js/views/coupon_form_view.js:545]: You can DRY this by applying polymorphism. Common code should reside in the base function.

**Comment by srpearce** [comment]
👍

**Comment by marjev** [comment]
👍

**Pull request merged**